content/handbook/company/culture/inclusion/_index.md

```
@@ -1,282 +1,282 @@
-コンテキストは「GitLabにおいて」または「GitLabの特定の部門またはチームにおいて」です。
-この用語は一般的に、GitLabで働くことを望む、または現在働いている人々のソーシング、インタビュー、採用、定着を理解し改善するためにGitLabがどのように取り組んでいるかについての報告のコンテキストで使用されます。
+文脈は「GitLab において」または「GitLab の特定の部門やチームにおいて」です。
+この用語は通常、GitLab で働きたいまたは現在働いている人々のソーシング、面接、採用、定着について、GitLab がどのように理解し改善に取り組んでいるかを報告する文脈で使用されます。
 
-* 個人を「多様な人物」または「多様性採用」と呼ぶ**べきではなく**、それは彼らが現在のコミュニティに含まれていないことを示唆したり、彼らが直接スキルや仕事の能力に関係しない要因でのみ採用されているように見せることになります。
-* 個人的なインタラクションで冷たい用語のラベルで人を特別視したり「他者化」したりすべきではありません。
+* 1 人の人物を「diverse person (多様な人)」または「diversity hire (多様性採用)」と言ってはいけません。これは、彼らが現在のコミュニティに含まれていないこと、または彼らが自分のスキルや仕事をする能力に直接関係しない要因のためだけに雇われていることを意味してしまいます。
+* 個人的な対話の中で、冷たい用語のラベルで人を区別したり「他者化」したりすべきではありません。
 
-GitLabがこのデータを使用して進展を図る方法についての詳細は、["過小代表グループの選択例"](#examples-of-select-underrepresented-groups)をご覧ください。
+GitLab がこのデータをどのように使用して進歩しているかについての追加情報は、["代表性不足グループの選定"](#examples-of-select-underrepresented-groups) を参照してください。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)